seumich/tests.py

```diff
@@ -572,17 +571,17 @@ def test_student_class_site_view(self):
         self.assertQuerysetEqual(response.context['advisors'],
                                  [('<StudentCohortMentor: grace is in the '
                                    'Special Probation F14 cohort>')])
-        self.assertQuerysetEqual(response.context['scoreData'],
-                                 [("{'color': '#255c91', 'values': [[1, 0L], "
-                                   "[2, 0L], [3, 0L], [4, 0L], [5, 65L], "
-                                   "[6, 68L], [7, 68L], [8, 68L], [9], [10], "
-                                   "[11], [12], [13], [14]], "
-                                   "'key': 'Student'}"),
-                                  ("{'color': '#F0D654', 'values': [[1, 0L], "
-                                   "[2, 0L], [3, 0L], [4, 0L], [5, 58L], "
-                                   "[6, 58L], [7, 58L], [8, 57L], [9], [10], "
-                                   "[11], [12], [13], [14]], 'key': 'Class'}")
-                                  ])
```

Comment:
I'm not crazy about including the commented out line in committed code, but I'll defer to you. If you remove it, let's also remove `import sys`, as that's throwing a Codacy error.

Reply:
Ok yeah I'll remove it. Hopefully we'll remember if we ever need to try to debug this in the future. Maybe we can add a better way to debug running these unit tests, like using ptvsd or something too.